generator.json

```diff
@@ -2,33 +2,203 @@
+          "!sources/**/DirectXHelpers/DirectX.Manual.cs": "!**/DirectXHelpers/DirectX.Manual.cs",
+          "!sources/**/d2d1/DirectX.Manual.cs": "!**/d2d1/DirectX.Manual.cs",
+          "!sources/**/IWeakReference.Manual.cs": "!**/IWeakReference.Manual.cs",
+          "!sources/**/IDispatchEx.Manual.cs": "!**/IDispatchEx.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM_PARSE_HELPER.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM1_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM1_PARSE_HELPER.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM2_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM2_PARSE_HELPER.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM3_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM3_PARSE_HELPER.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM4_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM4_PARSE_HELPER.Manual.cs",
+          "!sources/**/CD3DX12_PIPELINE_STATE_STREAM5_PARSE_HELPER.Manual.cs": "!**/CD3DX12_PIPELINE_STATE_STREAM5_PARSE_HELPER.Manual.cs",
+          "!sources/**/ShObjIdl_core/Windows.Manual.cs": "!**/ShObjIdl_core/Windows.Manual.cs",
+          "!sources/**/propvarutil/Windows.Manual.cs": "!**/propvarutil/Windows.Manual.cs",
+          "!sources/**/uuids/Windows.Manual.cs": "!**/uuids/Windows.Manual.cs",
+          "!sources/**/gdipluscolor/Color.Manual.cs": "!**/gdipluscolor/Color.Manual.cs",
+          "!sources/**/windows.ui.input/IID.Manual.cs": "!**/windows.ui.input/IID.Manual.cs",
+          "!sources/**/windows.security.authentication.web.core/IID.Manual.cs": "!**/windows.security.authentication.web.core/IID.Manual.cs",
+          "!sources/**/roapi/WinRT.Manual.cs": "!**/roapi/WinRT.Manual.cs",
+          "!sources/**/windows.graphics.directx.direct3d11.interop/WinRT.Manual.cs": "!**/windows.graphics.directx.direct3d11.interop/WinRT.Manual.cs",
+          "!sources/**/roparameterizediid/**.Manual.cs": "!**/roparameterizediid/**.Manual.cs",
+          "!sources/**/d2d1_1/D2D1_**.Manual.cs": "!**/d2d1_1/D2D1_**.Manual.cs"
+        }
+      },
+      "ChangeNamespace": {
+        "Mappings": {
+          "TerraFX.Interop.Windows(.*)": "Silk.NET.Win32$1",
+          "TerraFX.Interop(?!.Windows)(.*)": "Silk.NET$1"
+        }
+      },
+      "TransformInterfaces": {
+        "AdditionalInterfaces": [
+          "IDispatchEx",
+          "IUnknown",
+          "IInspectable",
+          "IClassFactory",
+          "AsyncIUknown"
+        ]
+      },
+      "TransformCOM": {
+        "BaseTypes": {
+          "IUnknown.Interface": "Silk.NET.Core.IUnknown.Interface"
+        },
+        "AdditionalCOMTypes": {
+          "IDispatchEx.Interface": {
+            "Namespace": "Silk.NET.Win32",
+            "Parent": "IDispatch.Interface"
+          },
+          "IInspectable.Interface": {
```

Comment:
Tbh, all the ones that are in Silk.NET.Core should be in additional COM types by default.

Reply:
fair, we can append them easily enough.

Reply:
fixed